components/menu/index.css

```diff
@@ -10,170 +10,676 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+
+  /* "one" icon: chevron + additional icon (we don't count the chevron as an icon because it HAS to be there for a collapisble) */
+  --spectrum-menu-item-collapisble-has-icon-submenu-item-padding-x-start:
+    calc((
+      var(--spectrum-menu-item-label-inline-edge-to-content)
+        + var(--spectrum-menu-item-checkmark-width)
+        + var(--spectrum-menu-item-text-to-control)
+        + var(--spectrum-menu-item-icon-width)
+        + var(--spectrum-menu-item-label-text-to-visual)
+    ));
+
+  /* "no" icon: just the chevron (we're not counting it because it HAS to be there for a collapisble) */
+  --spectrum-menu-item-collapisble-no-icon-submenu-item-padding-x-start:
+    calc((
+      var(--spectrum-menu-item-label-inline-edge-to-content)
+        + var(--spectrum-menu-item-checkmark-width)
+        + var(--spectrum-menu-item-label-text-to-visual)
+        - var(--spectrum-menu-item-focus-indicator-width)
+    ));
+}
+
+@media (forced-colors: active) {
+  .spectrum-Menu {
+    .spectrum-Menu-item,
+    .spectrum-Menu-item--collapsible,
+    .spectrum-Menu-item--drillIn {
+      forced-color-adjust: none;
+
+      --highcontrast-menu-item-label-content-color-default: ButtonText;
+      --highcontrast-menu-item-description-color-default: ButtonText;
+      --highcontrast-menu-section-header-color: ButtonText;
+      --highcontrast-menu-item-value-color-default: ButtonText;
+      --highcontrast-menu-item-value-color-default: ButtonText;
+      
+      border-color: HighlightText;
+
+      &:hover,
+      &:active {
+        --highcontrast-menu-item-label-content-color-hover: HighlightText;
+        --highcontrast-menu-item-label-content-color-down: HighlightText;
+        
+        --highcontrast-menu-item-label-icon-color-hover: HighlightText;
+        --highcontrast-menu-item-label-icon-color-down: HighlightText;
+        
+        --highcontrast-menu-item-description-color-hover: HighlightText;
+        --highcontrast-menu-item-description-color-down: HighlightText;
+        
+        --highcontrast-menu-section-header-color: HighlightText;
+        --highcontrast-menu-collapsible-icon-color: HighlightText;
+        
+        --highcontrast-menu-checkmark-icon-color-hover: HighlightText;
+        --highcontrast-menu-checkmark-icon-color-down: HighlightText;
+        
+        --highcontrast-menu-drillin-icon-color-hover: HighlightText;
+        --highcontrast-menu-drillin-icon-color-down: HighlightText;
+        
+        --highcontrast-menu-item-background-color-hover: Highlight;
+        --highcontrast-menu-item-background-color-down: Highlight;
+        --highcontrast-menu-item-background-color-key-focus: Highlight;
+
+        --highcontrast-menu-item-value-color-hover: HighlightText;
+        --highcontrast-menu-item-value-color-down: HighlightText;
+        --highcontrast-menu-item-value-color-focus: HighlightText;
+      }
+      
+      &:focus-ring,
```

Comment:
Yes, there is a bug causing the `spectrum-Menu-itemLabel` text color to resolve as black when the `focus-ring` class is applied.

`focus-ring` can also be replaced with `:focus-visible` [source](https://lists.w3.org/Archives/Public/www-style/2017Dec/0018.html)

<img width="190" alt="Screen Shot 2022-09-22 at 3 23 59 PM" src="https://user-images.githubusercontent.com/16216104/191845012-6bec1817-d8d4-4d00-8286-c22c8929a73c.png">

<img width="360" alt="Screen Shot 2022-09-22 at 3 28 41 PM" src="https://user-images.githubusercontent.com/16216104/191844810-9d4a912e-b47f-45e6-9362-82c2022fd63e.png">

Reply:
Thanks! I fixed these up.